data/group.json

```diff
@@ -1,6 +1,6 @@
 {
   "A": ["Qatar", "Ecuador", "Senegal", "Netherlands" ],
-  "B": ["England", "Iran", "United_States", "Unknown"],
+  "B": ["England", "Iran", "USA", "Unknown"],
   "C": ["Argentina", "Saudi_Arabia", "Mexico", "Poland"],
   "D": ["France", "Unknown", "Denmark", "Tunisia"],

```

Comment:
We have 3 teams with the same name **Unknown**.
It is proposed to rename them to Unknown_**B**, Unknown_**D** and Unknown_**E**.
National teams should have similar names in the _schedule.json_ file. It needs to be corrected.